Complete switch cases

When processing `TransportLayerCommandType` with a `switch` statement, we should include cases that handle all possible values.

This can be done by adding a `default` section. It could also be done by explicitly mentioning unused `enum` values, using an empty block.

Using `default`:
```cpp
	case tlcRequestExternalAddress:
		ProcessRequestExternalAddress(packet, from);
		return;
	default: // Other messages are not relevant to the game server
		return;
	}
```

Using explicit case values:
```cpp
	case tlcRequestExternalAddress:
		ProcessRequestExternalAddress(packet, from);
		return;
	// Message types not relevant to the game server
	case tlcJoinHelpRequest:
	case tlcEchoExternalAddress:
		return;
	}
```

There is a slight difference between the two approaches. Using the `default` case, the compiler would not warn about missing case labels, even if `TransportLayerCommandType` is extended to have new values. The new values may be relevant to the switch block, and may need matching handler code added. Using explicit case labels ensures the compiler will warn if `TransportLayerCommandType` is extended with new values and they aren't explicitly handled by the `switch`.

----

GCC-8 will warn about missing `case` labels:
```
GameServer.cpp: In member function ‘void GameServer::ProcessPacket(Packet&, sockaddr_in&)’:
GameServer.cpp:178:8: warning: enumeration value ‘tlcJoinHelpRequest’ not handled in switch [-Wswitch]
  switch(packet.tlMessage.tlHeader.commandType)
        ^
GameServer.cpp:178:8: warning: enumeration value ‘tlcEchoExternalAddress’ not handled in switch [-Wswitch]
```

--- a/GameServer.cpp
+++ b/GameServer.cpp
@@ -203,6 +203,11 @@ void GameServer::ProcessPacket(Packet &packet, sockaddr_in &from)
 	case tlcRequestExternalAddress:
 		ProcessRequestExternalAddress(packet, from);
 		return;
+	// Message types not relevant to the game server
+	// Unused messages included to prevent GCC-8 case label warning
+	case tlcJoinHelpRequest:
+	case tlcEchoExternalAddress:
+		return;
 	}
 }
 

